### ISSUE TITLE
Move descriptions and status into free radius database

### ISSUE THREAD

**Issue by 1995parham** [aut-cic/internet#167]
Right now we are reading descriptions and status from files, but they need to have a way to manage it. The easiest way is to load this information from FreeRadius Database.

**Comment by armangiter** [comment]
you just need to set up a database using a framework like Django and you will be able to manage your data easily.

**Comment by 1995parham** [comment]
Yeah, but the project scale is not that big to have its own database. we are using sql alchemy and radius database.
